src/components/3d/SimpleModelView.tsx

```diff
@@ -7,9 +7,9 @@ import {
   ViewStyle,
 } from 'react-native';
 import { WebView } from 'react-native-webview';
-import styled from 'styled-components';
 import { ImgixImage } from '@rainbow-me/images';
 import { padding, position } from '@rainbow-me/styles';
+import styled from 'rainbowed-components';
```

Comment:
Should this be under the `@rainbow-me` scope to make it a bit clearer that it's internal?

Also, if we do that, I wonder if it's better to just call it `@rainbow-me/styled-components` since it's trying to be a drop-in replacement (with missing features, obviously) and the scope makes it clear it's our fork of it.